packages/firmware-release-config/src/index.ts

```diff
@@ -54,8 +53,8 @@ export const getFirmwareReleaseConfig = async () => {
     }
 };
 
-export const getReleasesMessage = async () => {
-    const { releasesJws } = await getFirmwareReleaseConfig();
+export const getFirmwareReleaseConfig = async () => {
```

Comment:
this interanlly uses "isCodesignBuild" utility and is called from inside connect (parseFirmwareReleaseConfig). Are ou sure that this env variable is set correctly for all thinkable environments where connect might be used?

Reply:
It should be available in suite-web and suite-desktop both environments where it is expected to be used.

It is also available in iframe (https://github.com/trezor/trezor-suite/blob/develop/packages/connect-iframe/webpack/base.webpack.config.ts#L112) but I do not think it is required there.